data-access/src/functions/http-graphql/schema/types/community.graphql

```diff
@@ -13,6 +13,21 @@ type Community implements MongoBase {
   createdAt: DateTime
   updatedAt: DateTime
   userIsAdmin: Boolean
```

Comment:
Update the community.resolvers.ts file to include approvedVendors under the Community type and resolve it like shown belown. Use VendorUser instead of Entity
![image](https://github.com/user-attachments/assets/e4f6b173-1c68-4c11-9af0-a83400604591)